tests/wpt/web-platform-tests/xhr/abort-after-send.any.js

```diff
@@ -9,7 +9,9 @@
         client.addEventListener("readystatechange", test.step_func(function() {
           if(client.readyState == 4) {
             control_flag = true
-            assert_equals(client.responseXML, null)
+            if (test.GLOBAL.isWindow()) {
```

Comment:
I think you can fix the failure by using `self.GLOBAL.isWindow()` 👀

Ref: https://web-platform-tests.org/writing-tests/testharness.html#multi-global-tests

Reply:
Oh ok, I was thinking it was referring to the actual name used for the test object.